test/unit/math/rev/mat/prob/test_gradients.hpp

```diff
@@ -2,6 +2,12 @@
 #include <vector>
 #include <iomanip>
 
+void test_grad_eq(Eigen::Matrix<double, -1, 1> grad_1,
+                    Eigen::Matrix<double, -1, 1> grad_2) {
+  for (int i = 0; i < grad_1.size(); ++i) 
```

Comment:
@rtrangucci, add a quick check for size grad_1 == size grad_2 if you want to reuse this function.